cmd/kube-runner/go-runner.go

```diff
@@ -0,0 +1,123 @@
+/*
+Copyright 2020 The Kubernetes Authors.
```

Comment:
This is the original go-runner.go from @dims. I kept the file name, but perhaps I should better rename also it to kube-runner.go?